packages/@aws-cdk/aws-appmesh/test/test.virtual-service.ts

```diff
@@ -34,6 +36,90 @@ export = {
     // THEN
     test.equal(virtualService.mesh.meshName, meshName);
     test.equal(virtualService.virtualServiceName, virtualServiceName);
+
     test.done();
   },
+
+  'When adding a VirtualService to a mesh': {
```

Comment:
Moved these methods from the `test.mesh.ts` file. Did not modify them beyond changing `mesh.addVirtualService(...)` to `new appmesh.VirtualService(...)`